blocks/v2-columns/v2-columns.css

```diff
@@ -20,6 +20,12 @@
   display: block;
 }
 
+.v2-columns__column--with-text .icon,
```

Comment:
Btw mobile and tablet, it looks really strange:
<img width="695" alt="image" src="https://github.com/Netcentric/vg-macktrucks-com-rd/assets/3090999/4183a559-eac2-4439-a890-a731e7751296">

Reply:
In tablet view, I would say, the most important element is the smallest, is that expected @cogniSyb ?
<img width="705" alt="image" src="https://github.com/Netcentric/vg-macktrucks-com-rd/assets/3090999/bfc9068a-29ee-4662-a8f1-5e56d600aa34">

Reply:
Now they have the same width